In-house supporting volume_dd_blocksize ('1M', 'G') to enable 'dd' to
run on Solaris. This patch is Solaris-specific and not suitable for
upstream.

--- cinder-2015.1.2/cinder/volume/utils.py.orig	2015-11-17 18:29:59.168650488 -0800
+++ cinder-2015.1.2/cinder/volume/utils.py	2015-11-17 18:30:21.197972544 -0800
@@ -267,6 +267,7 @@
         bs = strutils.string_to_bytes('%sB' % blocksize)
 
     count = math.ceil(size_in_m * units.Mi / bs)
+    blocksize = int(bs)
 
     return blocksize, int(count)
 
